provider/blob/src/export/mod.rs

```diff
@@ -23,7 +23,7 @@
 //! DatagenDriver::new()
 //!     .with_keys([icu_provider::hello_world::HelloWorldV1Marker::KEY])
 //!     .with_all_locales()
-//!     .export(&DatagenProvider::new_latest_tested(), exporter)
+//!     .export(&icu_provider::hello_world::HelloWorldProvider, exporter)
```

Comment:
This change doesn't impact behavior; it just makes it so that this crate doesn't need to depend on the `provider` feature of icu_datagen